templates/catalog/_partials/miniatures/pack-product.tpl

```diff
@@ -35,6 +35,8 @@
                   {if !empty($product.default_image.medium.sources.webp)}<source srcset="{$product.default_image.medium.sources.webp}" type="image/webp">{/if}
                   <img
                           src="{$product.default_image.medium.url}"
+                          width="{$product.default_image.bySize.medium_default.width}"
+                          height="{$product.default_image.bySize.medium_default.height}"
```

Comment:
```suggestion
                          width="{$product.default_image.medium.width}"
                          height="{$product.default_image.medium.height}"
```

Reply:
@buggyzap Please use the medium image, it could resolve into something different, theoretically.